Bug(Util): find_missing_translations.js is not working (on Linux?)

# Description

Whether I call the main function manually or call it via npm script, the file just prints files not found error and exit.

Here is the terminal output when I was using the npm script way.

```bash
$ npm run find-translations -- --locale=ja --filter=e9s

> cactbot@ find-translations /workspaces/cactbot-workspace/cactbot
> node --loader ts-node/esm util/find_missing_translations.js "--locale=ja" "--filter=e9s"

(node:46215) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
Error: ENOENT: no such file or directory, open '\workspaces\cactbot-workspace\cactbot\ui\oopsyraidsy\data\05-shb\raid\e9s.js'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cactbot@ find-translations: `node --loader ts-node/esm util/find_missing_translations.js "--locale=ja" "--filter=e9s"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cactbot@ find-translations script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-06-26T13_28_34_321Z-debug.log
```

I have tried to call the main function manually (by my new CLI), it output the same error.

```bash
$ npm run util

> cactbot@ util /workspaces/cactbot-workspace/cactbot
> node --loader ts-node/esm util/index.ts

(node:49642) ExperimentalWarning: --experimental-loader is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
? What do you want to do? Find Raidboss missing translations
? Input a valid trigger JavaScript filename:  ui/raidboss/data/05-shb/raid/e9s.js
? Select a locale:  ja
Error: ENOENT: no such file or directory, open '\workspaces\cactbot-workspace\cactbot\ui\raidboss\data\05-shb\raid\e9s.js'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cactbot@ util: `node --loader ts-node/esm util/index.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cactbot@ util script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2021-06-26T13_47_05_870Z-debug.log
```

### Log

<details><summary>npm log file content (Click to Expand)</summary>
<pre>
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/bin/node',
1 verbose cli   '/usr/local/share/npm-global/bin/npm',
1 verbose cli   'run',
1 verbose cli   'find-translations',
1 verbose cli   '--',
1 verbose cli   '--locale=ja',
1 verbose cli   '--filter=e9s'
1 verbose cli ]
2 info using npm@6.14.13
3 info using node@v14.15.4
4 verbose run-script [
4 verbose run-script   'prefind-translations',
4 verbose run-script   'find-translations',
4 verbose run-script   'postfind-translations'
4 verbose run-script ]
5 info lifecycle cactbot@~prefind-translations: cactbot@
6 info lifecycle cactbot@~find-translations: cactbot@
7 verbose lifecycle cactbot@~find-translations: unsafe-perm in lifecycle true
8 verbose lifecycle cactbot@~find-translations: PATH: /usr/local/share/npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/workspaces/cactbot-workspace/cactbot/node_modules/.bin:/vscode/vscode-server-insiders/bin/x64/9744231fc12f1aed21089180b3f0394d694bd2a2-insider/bin:/usr/local/share/npm-global/bin:/usr/local/share/nvm/current/bin:/usr/local/share/npm-global/bin:/usr/local/share/nvm/current/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/node/.local/bin
9 verbose lifecycle cactbot@~find-translations: CWD: /workspaces/cactbot-workspace/cactbot
10 silly lifecycle cactbot@~find-translations: Args: [
10 silly lifecycle   '-c',
10 silly lifecycle   'node --loader ts-node/esm util/find_missing_translations.js "--locale=ja" "--filter=e9s"'
10 silly lifecycle ]
11 silly lifecycle cactbot@~find-translations: Returned: code: 1  signal: null
12 info lifecycle cactbot@~find-translations: Failed to exec find-translations script
13 verbose stack Error: cactbot@ find-translations: `node --loader ts-node/esm util/find_missing_translations.js "--locale=ja" "--filter=e9s"`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/local/share/npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/local/share/npm-global/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1048:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid cactbot@
15 verbose cwd /workspaces/cactbot-workspace/cactbot
16 verbose Linux 5.10.25-linuxkit
17 verbose argv "/usr/local/bin/node" "/usr/local/share/npm-global/bin/npm" "run" "find-translations" "--" "--locale=ja" "--filter=e9s"
18 verbose node v14.15.4
19 verbose npm  v6.14.13
20 error code ELIFECYCLE
21 error errno 1
22 error cactbot@ find-translations: `node --loader ts-node/esm util/find_missing_translations.js "--locale=ja" "--filter=e9s"`
22 error Exit status 1
23 error Failed at the cactbot@ find-translations script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
</pre>
</details>

### VSCode About
Version: 1.58.0-insider (user setup)
Commit: 9744231fc12f1aed21089180b3f0394d694bd2a2
Date: 2021-06-23T05:13:08.071Z
Electron: 12.0.11
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.19041

### And I work on a Linux machine
OS: Debian 10 buster
Shell: bash 5.0.3
NodeJS: 14.15.4
NPM: 6.14.13
Yarn: 1.22.5

--- a/util/find_missing_translations.js
+++ b/util/find_missing_translations.js
@@ -55,8 +55,8 @@ const findAllJavascriptFiles = (filter) => {
       return;
     if (filter !== undefined && !filepath.includes(filter))
       return;
-    // These are full paths, so use backslashes to match Windows path names.
-    arr.push(filepath.replace(/\//g, '\\'));
+    // normalize file path, handles windows vs linux separators
+    arr.push(path.normalize(filepath));
   });
   return arr;
 };

